verification/backends/email.py

```diff
@@ -2,24 +2,37 @@
 from __future__ import absolute_import
 
 # Third party
-from django.core import mail
+from django.template import Context
+from django.template import loader
+from django.core.mail import EmailMultiAlternatives
 
 # Local
 from .base import BaseBackend
 
 
 class EmailBackend(BaseBackend):
+    html_template = 'verification/email.html'
+    subject = 'Please verify your email'
+
     def __init__(self, **options):
         super(EmailBackend, self).__init__(**options)
         # Lower case it just to be sure
         options = {key.lower(): value for key, value in options.items()}
-        self.from_email = options.get('from_email')
-
-    def send(self, emails, subject, message, html_message=None):
-        delivered_messages = mail.send_mail(
-                                       from_email=self.from_email,
-                                       subject=subject,
-                                       message=message,
-                                       fail_silently=False,
-                                       recipient_list=emails)
-        return delivered_messages
+        self.from_email = options.get('from_email', None)
+        self.html_template = options.get('html_template', self.html_template)
+        self.subject = options.get('subject', self.subject)
+
+    def send(self, email, link):
+        html = self._create_html(link)
```

Comment:
thoughts on changing the var name and template to just handle a generic `context`?